mempool/src/pool/tests/mod.rs

```diff
@@ -199,6 +205,12 @@ pub async fn start_chainstate_with_config(
     start_chainstate(chainstate).await
 }
 
+pub async fn start_chainstate_with_config(
+    chain_config: Arc<ChainConfig>,
+) -> subsystem::Handle<Box<dyn ChainstateInterface>> {
+    start_chainstate_with_config_and_max_tip_age(chain_config, MaxTipAge::CENTURY).await
+}
```

Comment:
It's better to rename this function, because it modifies the passed config in a specific way.

But in general I'd prefer to leave the original `start_chainstate_with_config` as it was, and just pass different configs into it depending on the needs of each particular test.

Reply:
Ah, I see.
But anyway, this function uses some non-default value for a config parameter that has a specific purpose (preventing the IBD state), so IMO it deserves having a specific name.

I don't insist on it though.

Reply:
> Ah, I see. But anyway, this function uses some non-default value for a config parameter that has a specific purpose (preventing the IBD state), so IMO it deserves having a specific name.

I did not end up changing the name, basically for two reasons:
* I did not find a name that would be both sufficiently descriptive and concise. Instead, I added a comment pointing out it uses some non-default values, which I think is a reasonable compromise.
* Vast majority of mempool tests (all except ones that specifically test IBD handling) rely on IBD being disabled. So in context of mempool testing, the huge max tip age is de-facto default anyway.

Also the two versions of the `start_chainstate_with_*` method are a residue of the development that is no longer necessary, so there's now only one version.